austation/code/game/gamemodes/bloodsucker/bloodsucker.dm

```diff
@@ -0,0 +1,316 @@
+	false_report_weight = 1
+	restricted_jobs = list("AI","Cyborg")
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel", "Brig Physician")
+	required_players = 12
+	required_enemies = 1
+	recommended_enemies = 4
+	reroll_friendly = 1
+	enemy_minimum_age = 7
+
+	announce_span = "danger"
+	announce_text = "Filthy, bloodsucking vampires are crawling around disguised as crewmembers!\n\
+	<span class='danger'>Bloodsuckers</span>: The crew are cattle, while you are both shepherd and slaughterhouse.\n\
+	<span class='notice'>Crew</span>: Put an end to the undead infestation before the station is overcome!"
+
+
+
+// Seems to be run by game ONCE, and finds all potential players to be antag.
+/datum/game_mode/bloodsucker/pre_setup()
+
+	// Set Restricted Jobs
+	if(CONFIG_GET(flag/protect_roles_from_antagonist))
+		restricted_jobs += protected_jobs
+
+	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
+		restricted_jobs += "Assistant"
+
+	// Set number of Vamps
+	recommended_enemies = clamp(round(num_players()/10), 1, 6);
+
+	// Select Antags
+	if(antag_candidates.len>0)
+		for(var/i = 0, i < recommended_enemies, i++)
+			if (!antag_candidates.len)
+				break
+			var/datum/mind/bloodsucker = antag_pick(antag_candidates, ROLE_BLOODSUCKER)
+			//Can we even BE a bloodsucker?
+			if (can_make_bloodsucker(bloodsucker, display_warning=FALSE))
+				bloodsuckers += bloodsucker
+				bloodsucker.special_role = ROLE_BLOODSUCKER
+				bloodsucker.restricted_roles = restricted_jobs
+				log_game("[bloodsucker.key] (ckey) has been selected as a Bloodsucker.")
+				antag_candidates.Remove(bloodsucker) // Apparently you can also write antag_cndidates -= bloodsucker
+		return 1
+	else
+		setup_error = "Not enough bloodsucker candidates"
+		return 0
+
+	// FULPSTATION: Assign Hunters (as many as monsters, plus one)
+	//assign_monster_hunters(bloodsuckers.len, TRUE, bloodsuckers)	// austation -- DISABLING HUNTERS FROM ROUND START, We have enough validhunters anyway
+
+	// Do we have enough vamps to continue?
+	// return bloodsuckers.len >= required_enemies
+
+
+// Gamemode is all done being set up. We have all our Vamps. We now pick objectives and let them know what's happening.
+/datum/game_mode/bloodsucker/post_setup()
+
+	// Sunlight (Creating Bloodsuckers manually will check to create this, too)
+	check_start_sunlight()
+
+	// Vamps
+	for(var/datum/mind/bloodsucker in bloodsuckers)
+		// spawn() --> Run block of code but game continues on past it.
+		// sleep() --> Run block of code and freeze code there (including whoever called us) until it's resolved.
+
+		//Clean Bloodsucker Species (racist?)
+		//clean_invalid_species(bloodsucker)
+		// 			TO-DO !!!
+
+		// Add Bloodsucker Antag Datum (or remove from list on Fail)
+		if (!make_bloodsucker(bloodsucker))
+			bloodsuckers -= bloodsucker
+			bloodsucker.special_role = null
+
+	// NOTE: Hunters are done in ..() parent proc
+
+	return ..()
+
```

Comment:
```suggestion

/datum/game_mode/bloodsucker/generate_report()
	return "Persisting rumours about ancient lineages of haematophagic undeads (i. e. vampires) entered a new phase, and there are now hard-to-dispel rumours about vampires trying to expand their reaches into space. \
			Investigate whether there is any sign of vampiric activities onboard the station."
			
```
People, come to have a look at this, for no one actually gives a shit on these.

The reason why I am doing this; popped up at one of my testing session:
![Screenshot 2020-09-19 204540](https://user-images.githubusercontent.com/8010007/93693643-815aa680-fb3d-11ea-8ba9-ecce45e83f7d.png)

Reply:
![Screenshot 2020-09-20 123908](https://user-images.githubusercontent.com/8010007/93693705-53c22d00-fb3e-11ea-8f63-1ec791be9a49.png)
![Screenshot 2020-09-20 123918](https://user-images.githubusercontent.com/8010007/93693707-558bf080-fb3e-11ea-8799-eaf09ceff506.png)

Reply:
oh shit, legendary park man doing some more bug snapping. Good find